src/Symfony/Component/VarDumper/Dumper/HtmlDumper.php

```diff
@@ -891,7 +891,7 @@ protected function style($style, $value, $attr = [])
                 $link = '';
             }
 
-            return sprintf('<abbr title="%s" class=sf-dump-%s>%s</abbr>%s', $v, $style, substr($v, $c + 1), $link);
```

Comment:
The variable `$c` is now unused and should be removed.

Reply:
Thanks, It's removed

Reply:
Yes, but you might have removed a bit too much.